méli-mélo/méli-mélo-en.html

```diff
@@ -3,105 +3,157 @@
+	<li>initiate usability research;</li>
+	<li>initiate preliminary discussion with key organisation; and</li>
+	<li>transforming the feature into a high quality reusable product adapted for GCWeb.</li>
 </ul>
 
-<h2>Technical review checklist</h2>
+<h3 id="mli-mlo-list">Active méli-mélo compilations and their features</h3>
 
-<p>This list contains the steps that the wet-boew technical review team uses to approve new méli-mélo features.</p>
-<ul>
-	<li>Ensure the project sponsor is clearly identified;</li>
-	<li>Project folder name follows the naming convention explained above;</li>
-	<li>Each JavaScript feature and each style are demoed;</li>
-	<li>Perform a code review to ensure there are no overrides nor conflict with GCWEb and/or wet-boew;</li>
-	<li>Quickly check if we don't notice any major or obvious web accessibility and security issue. However, the accessibility conformance remains the responsibility of the publisher when implementing the feature into a web page;</li>
-	<li>Ensure the feature doesn't automatically change the state of a generic page when loaded. It must not impact any content unless the feature is explicitly activated through the HTML code, like through the use of a CSS class or the presence of a data attribute;</li>
-	<li>Review the implementation plan to ensure it contains a reasonable due date and deliverable. It must include:
-		<ul>
-			<li>Engagement with the Digital Transformation Office (DTO) at Treasury Board Secretariat;</li>
-			<li>Review and perform the identification of the feature transformation requirement to be able to complete the integration progress into GCWeb;</li>
-			<li>Produce accessibility conformance report and attach usability report, if any;</li>
-			<li>Transformation of the méli-mélo functionality into a GCWeb provisional feature;</li>
-			<li>Complete feature stabilisation task, like working example translation, writing guidance, publishing ACRs, feature API documentation, etc.</li>
-		</ul>
+<ul class="row list-unstyled wb-eqht-grd mrgn-tp-md">
+
+{% for item in site.data[ "mli-mlo" ].packages %}
+	<li class="col-xs-12 col-md-4 mrgn-tp-md mrgn-bttm-md">
+		<div class="brdr-tp brdr-rght brdr-bttm brdr-lft hght-inhrt">
+			<h4 class="mrgn-tp-md mrgn-rght-md mrgn-bttm-md mrgn-lft-md">{{ item.nom }}</h4>
+			<div class="mrgn-rght-md mrgn-bttm-md mrgn-lft-md">
+				<ul class="mrgn-bttm-lg mrgn-lft-md">
+				{% for pack in item.libs %}
+					{% assign indexPage = site.data[ "mli-mlo" ].subProjects | where: "nom", pack | first %}
+					<li><a href="/gcweb-compiled-demos/méli-mélo-demos/{{ item.nom }}/{{ pack }}/{{ indexPage.mainpage }}">{{ pack }}</a></li>
+				{% endfor %}
+				</ul>
+			</div>
+		</div>
 	</li>
+{% endfor %}
 </ul>
 
-<h2 id="mli-mlo-comp">Méli-mélo compilations</h2>
+<p><small>(<a href="compilation-gelé/index.html" hreflang="fr">View frozen and deprecated méli-mélo compilations (in French only)</a>.)</small></p>
```

Comment:
Something make the link broken

Reply:
I figured out that you only created a standalone page, you can ignore my comment